### PR TITLE
Update the nightly scripts

### PR DESCRIPTION
**Pull request by shubhamkumar13**
This PR updates the nightly bash scripts running in turing and navajo to 5.00.0+domains and 5.00.0+multicore

### CODE REVIEW

File: scripts/nightly_navajo.sh [hunk 1]
```diff
@@ -40,61 +42,77 @@ HOSTNAME="navajo"
...
-mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_412_DOMAINS_EFFECTS
-mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_412_STOCK
-mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_412_DOMAINS_EFFECTS
+#mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_MULTICORE
+mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_TRUNK
+mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_STABLE
+#mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_MULTICORE
+mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_TRUNK
+mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_STABLE
+#mkdir -p $SANDMARK_NIGHTLY_DIR/sandmark-nightly/pausetimes/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_412_STOCK/
```
Comment (shakthimaan) [inline scripts/nightly_navajo.sh:51]: Remove all unused and commented code.

**Comment by kayceesrk** [comment]
The original PR message 

> 5.00.0+domains and 5.00.0+multicore

seems incorrect. 

There are no `5.00.0+domains` and `5.00.0+multicore` anymore. Only `5.0+trunk` and `5.0+stable`. 

Also, in the OCaml developer meeting, it was decided that we would go with `5.0` instead of `5.00`. Perhaps useful to replace strings that are `5.00` to `5.0` .

File: scripts/nightly_turing.sh [hunk 2]
```diff
@@ -40,64 +41,88 @@ HOSTNAME="turing"
...
+	RUN_BENCH_TARGET=run_orunchrt \
 	BUILD_BENCH_TARGET=multibench_parallel \
 	RUN_CONFIG_JSON=multicore_parallel_run_config_filtered.json \
-	make ocaml-versions/4.12.0+domains+effects.bench
+	make ocaml-versions/5.00.0+stable.bench
 
-cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/4.12.0+domains+effects_1.orunchrt.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_412_DOMAINS_EFFECTS/
+cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/5.00.0+trunk_1.orunchrt.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_TRUNK/
+cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/5.00.0+stable_1.orunchrt.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_STABLE/
 rm -rf _results/
```
Comment (shakthimaan) [inline scripts/nightly_turing.sh:94]: Now that https://github.com/ocaml-bench/sandmark/pull/283 has been merged. Please add the folllowing to both the scripts:
```
SANDMARK_NIGHTLY_DIR=$SANDMARK_NIGHTLY_DIR bash run_all_custom.sh
```

**Comment by shubhamkumar13** [comment]
Closing this since these scripts would no longer be used.

**Pull request closed without merging**